mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/entity/EntityHelperGetMethodGenerator.java

```diff
@@ -89,7 +89,7 @@ public EntityHelperGetMethodGenerator(
           // category. Their getter creates a GenericType under the hood, so there's no performance
           // advantage in calling them instead of the generic get().
           getBuilder.addStatement(
-              "returnValue.$L(source.get($L, $T))",
+              "returnValue.$L(source.get($L, $L))",
```

Comment:
Noticed tests failing on 2.1 and it appears because I missed `setTags` so `tags` was never considered a property in my tests.  On fixing this I noticed helpers failed to compile because this expected a type, but really it should be a literal since `enclosingClass.addGenericTypeConstant(typeName)` adds and returns a type constant name.